Dockerfile

```diff
@@ -1,6 +1,9 @@
+FROM google/dart:2.5 as dart2
```

Comment:
Quick question, does this mean to run the tests in `Dart 2.5`?

Reply:
I had to add this line in Dockerfile, so would be able to use pub2 solver, otherwise pub get will timeout on dart1. 
Tests will run in dart2.5 tho, because we use `stable`: https://github.com/Workiva/r_tree/blob/master/.travis.yml#L4.